### PR TITLE
Implement cj command

### PR DESCRIPTION
**Pull request by quim0**
Related to #13969

### CODE REVIEW

File: libr/core/cmd_cmp.c [hunk 1]
```diff
@@ -245,6 +245,9 @@ static int radare_compare(RCore *core, const ut8 *f, const ut8 *d, int len, int
 	if (len < 1) {
 		return 0;
 	}
+    if (mode == 'j') {
+        r_cons_printf ("{\"diff_bytes\":[");
```
Comment (xarkes) [inline libr/core/cmd_cmp.c]: Better use the json API.
Check `libr/util/pj.c` and `pj_new`.

Comment (xarkes) [reply libr/core/cmd_cmp.c]: You can also find various usage examples in the codebase.

File: libr/core/cmd_cmp.c [hunk 2]
```diff
@@ -242,9 +242,19 @@ static int radare_compare_unified(RCore *core, ut64 of, ut64 od, int len) {
 
 static int radare_compare(RCore *core, const ut8 *f, const ut8 *d, int len, int mode) {
 	int i, eq = 0;
+    PJ *pj = NULL;
 	if (len < 1) {
 		return 0;
 	}
+    if (mode == 'j') {
```
Comment (radare) [inline libr/core/cmd_cmp.c]: use tabs

File: libr/core/cmd_cmp.c [hunk 3]
```diff
@@ -555,6 +580,18 @@ static int cmd_cmp(void *data, const char *input) {
 		free (str);
 	}
 	break;
+	case 'j':
+	{
+	    if (input[1] != ' ') {
```
Comment (radare) [inline libr/core/cmd_cmp.c]: i still see spaces here

File: libr/core/cmd_cmp.c [hunk 4]
```diff
@@ -555,6 +580,18 @@ static int cmd_cmp(void *data, const char *input) {
...
+	        eprintf ("Usage: cj [string]\n");
+	        return 0;
+	    }
```
Comment (radare) [inline libr/core/cmd_cmp.c]: maybe just } else instead of the early return?

File: libr/core/cmd_cmp.c (hunk 3 above)
Comment (quim0) [reply libr/core/cmd_cmp.c]: Oops, I'll fix it

**Pull request merged**